### DISCUSSION TITLE
Omnivore2trilium

### DISCUSSION THREAD

**Discussion by 0xbismarck** [TriliumNext/Trilium#6079] [Show and tell]
Hello:

I want to share with the community my new tool for exporting highlights from Omnivore, a Read-It-Later App, and sending the highlights to Trilium.
 - https://github.com/0xbismarck/omnivore2trilium

Assuming people find this tool useful, I'd be interesting in eventually turning this into a TriliumNext plugin.

~0xB

Crossposted:  [Trilium Discussion](https://github.com/zadam/trilium/discussions/4674)

**Comment by meichthys** [comment]
Neat! Hang on to that thought. It'll be a little while before we have a TriliumNext plugin system, but with an active developer community we should be able to get there!
